3604/1/table_output.rb

```diff
@@ -0,0 +1,39 @@
+require 'terminal-table'
+
+# output table with all information about gems
+class TableOutput
+  HASH_FOR_TABLE = {
+    name: '',
+    used_by: 'used by ',
+    watched_by: 'watched by ',
+    stars: 'stars ',
+    forks: 'forks ',
+    contributors: 'contributors ',
+    issues: 'issues '
+  }.freeze
+
+  def table_output(array_gems, count_gems_in_argv)
```

Comment:
I don't think it's a good idea to name method the same way as it's class. What about just `print`?